src/sbvr-api/sbvr-utils.d.ts

```diff
@@ -23,23 +23,26 @@ interface User {
 	permissions?: string[];
 }
 
+type HookReq = {
```

Comment:
Why are these types and not interfaces?

Reply:
Basically I copied how `HookArgs` is specified

Reply:
You added those too though? :P